app/blueprints/fund/routes.py

```diff
@@ -16,6 +16,7 @@
 from app.shared.table_pagination import GovUKTableAndPagination
 
 INDEX_BP_DASHBOARD = "index_bp.dashboard"
+FUND_TEMPLATE = "fund.html"
```

Comment:
I know sonarqube flagged duplication, but just ignore that one and keep referencing the file inline - I think this is fine.

Reply:
@samuelhwilliams i have reverted back now to use inline. we may need to pass this from Sonar